apis/managed-dimensions/lib/shapes/managed-dimension.ts

```diff
@@ -0,0 +1,87 @@
+import { dash, qudt, rdf, schema, sh, time, xsd } from '@tpluscode/rdf-ns-builders'
+import { supportedLanguages } from '@cube-creator/core/languages'
+import type { Initializer } from '@tpluscode/rdfine/RdfResource'
+import type { NodeShape } from '@rdfine/shacl'
+import $rdf from 'rdf-ext'
+import { meta, sh1 } from '@cube-creator/core/namespace'
+
+export default (): Initializer<NodeShape> => ({
```

Comment:
May I ask why you used a different way to declare the shape than in the main API?

Reply:
I decided not to put the shapes in the store, because we only have that single graph and that would be rather difficult to maintain.

So since they are initialised in code, it's easier to create them using JS rather turtle and parse